app/lib/katello/util/docker_manifest_clause_generator.rb

```diff
@@ -0,0 +1,44 @@
+module Katello
```

Comment:
Want to add some tests for this class, similar to the PackageClauseGeneratorTest?